README.md

```diff
@@ -331,6 +332,10 @@ The project uses [Sentry](https://sentry.io) to monitor errors. For development
 
 For production environment, you need to create a Sentry account and create a new project. Then, in `next.config.mjs`, you need to update the `org` and `project` attribute in `withSentryConfig` function. You also need to add your Sentry DSN in `sentry.client.config.ts`, `sentry.edge.config.ts` and `sentry.server.config.ts`.
 
+### Code coverage
+
+NextJS Boilerplate relies on [Codecov](https://codecov.io) for code coverage reporting solution. Create a Codecov account and connect it to your GitHub account. On your Codecov dashboard, it should display a list of your repositories. Select the repository you want to enable Codecov for and copy the token. Then, in your GitHub Actions, you need to define the `CODECOV_TOKEN` environment variable and paste the token you copied.
```

Comment:
Choosing to be really explicit on the Github Actions secret aspect. You can take it or leave it, but those new to github actions sometimes don't understand how env vars are used and end up pasting credentials (like the codecov token) directly into their github actions yaml file.

```suggestion
NextJS Boilerplate relies on [Codecov](https://codecov.io) for code coverage reporting solution. To use Codecov, Create a Codecov account and connect it to your GitHub account. On your Codecov dashboard, it should display a list of your repositories. Select the repository you want to enable Codecov for and copy the token. Then, in your GitHub Actions, you need to define the `CODECOV_TOKEN` environment variable and paste the token you copied. 

Be sure to create the `CODECOV_TOKEN` as a Github Actions secret, do not paste it directly into your source code.
```